src/Receipt/DonorDetailsGroup/Details/BillingAddress.php

```diff
@@ -0,0 +1,48 @@
+<?php
+namespace Give\Receipt\DonorDetailsGroup\Details;
+
+use Give\Receipt\Detail;
+use function give_get_donation_address as getDonationDonorAddress;
+
+/**
+ * Class BillingAddress
+ *
+ * @since 2.7.0
+ * @package Give\Receipt\DonorDetailsGroup\Details
+ */
+class BillingAddress extends Detail {
+	/**
+	 * @inheritDoc
+	 */
+	public function getLabel() {
+		return __( 'BILLING ADDRESS', 'give' );
```

Comment:
Because translated strings are user-defined, the translated string should be escaped.